### PR TITLE
[#101] Newest publication list on landing page should only contain it…

### PR DESCRIPTION
**Pull request by Possommi**
…ems being part of the statistics

### CODE REVIEW

**Comment by Possommi** [comment]
requires https://mycore.atlassian.net/browse/UBO-165 to be merged

File: common/src/main/resources/config/thunibib-common/messages_en.properties [hunk 1]
```diff
@@ -1,3 +1,6 @@
+facets.facet.origin                             = Structural unit of the university
```
Comment (kkrebs) [inline common/src/main/resources/config/thunibib-common/messages_en.properties:1]: only facets.facet.origin_exact needed here

File: jena/src/main/resources/META-INF/resources/index.xed [hunk 2]
```diff
@@ -94,29 +94,23 @@
...
           </button>
-
         </div>
       </xed:form>
     </div>
   </article>
 
-  <xinclude
-      uri="xslStyle:response-newest:solr:q=status:confirmed+AND+year:NOW&amp;rows=4&amp;sort=year+desc,created+desc&amp;facet=false&amp;fl=id,year"/>
+  <xinclude uri="xslStyle:response-newest:solr:q=%2BpartOf:true+status:confirmed+AND+year:NOW&amp;rows=4&amp;sort=year+desc,created+desc&amp;facet=false&amp;fl=id,year"/>
```
Comment (kkrebs) [inline jena/src/main/resources/META-INF/resources/index.xed:108]: should be:
xslStyle:response-newest:solr:q=partOf:true+AND+status:confirmed+AND+year:NOW&amp;rows=4&amp;sort=year+desc,created+desc&amp;facet=false&amp;fl=id,year

**Review by kkrebs — CHANGES_REQUESTED** [review]
see comments

**Pull request merged**